js/templates/election-cycles.hbs

```diff
@@ -0,0 +1,15 @@
+{{#each this}}
+  <label>
```

Comment:
Label should have a `for=[id of input]` attribute and then the `input` needs the same id.